index.bs

```diff
@@ -178,13 +178,10 @@ the user agent MUST run the following steps:
 
 1. If {{pictureInPictureElement}} is null, throw a {{InvalidStateError}} and abort
     these steps.
-2. Let |pipWindow| be the current Picture-in-Picture window.
-3. Let |pipVideo| be the video contained in the |pipWindow|.
-4. Let |video| be the source video of |pipVideo| frames.
-5. Close |pipWindow|.
-6. Render |pipVideo| frames in the |video|.
-7. Unset {{pictureInPictureElement}}.
-8. <a>Queue a task</a> to <a>fire an event</a> with the name
+2. Set the <a>Picture-in-Picture window</a> associated with
+    {{pictureInPictureElement}} |state| to |closed|.
```

Comment:
What about creating something called "close" and then say that when a PIPWindow is closed, its state switches to close. I think it will have the benefit of keeping "state" internal and allowing to easily expand what "close" means.

Reply:
Done.